games/src/schema/platforms.ts

```diff
@@ -0,0 +1,37 @@
+import { DistributionPlatform } from "../models";
+
+interface PlatformChoice {
+  value: DistributionPlatform;
+  name: string;
+}
+
+export const PLATFORM_CHOICES: PlatformChoice[] = [
+  {
+    value: "steam",
+    name: "Steam",
+  },
+  {
+    value: "steam-direct",
+    name: "Steam (bypass Steam for launching the game)",
+  },
+  {
+    value: "epic-games-store",
+    name: "Epic Games Store",
+  },
+  {
+    value: "oculus-store",
+    name: "Oculus Store",
+  },
+  {
+    value: "origin",
+    name: "Origin",
```

Comment:
Origin was replaced by EA Desktop ~2 years ago. Continuing to use the `origin` platform identifier is fine, however the `name` field here should probably be "EA Desktop" or "EA Desktop / Origin".